C/CUDA/CUDA_full@10.2/build_tarballs.jl

```diff
@@ -18,6 +18,19 @@ sources_windows = [
                "b538271c4d9ffce1a8520bf992d9bd23854f0f29cee67f48c6139e4cf301e253", "installer.exe")
 ]
 
+version_linux_aarch64 = v"10.2.300"
+sources_linux_aarch64 = [
+    AptSource(
+        "https://repo.download.nvidia.com/jetson/common",
+        "r32.6",
+        ["main"],
+        "http://repo.download.nvidia.com/jetson/jetson-ota-public.asc",
+        [
+            "cuda-toolkit-10-2"
+        ],
+        "cuda-toolkit-10-2")
+]
```

Comment:
I'd really like some guarantee that this file won't change on us.  E.g. embedding some kind of hash here would be helpful.

Reply:
I completely agree. I guess what one could do would be to be more specific about the versions of the mentioned APT packages, and then list both their version and their expected hash, e.g. (also renaming AptSource to AptPkgSource - to add list of AptSource's and list of AptKey's):
```julia
    AptPkgSource(
        [
            AptSource("https://repo.download.nvidia.com/jetson/common", "r32.6", ["main"])
        ],
        [
            AptKey("http://repo.download.nvidia.com/jetson/jetson-ota-public.asc"),
        ],
        [
            AptPkgSpec("cuda-toolkit-10-2", v"10.2.460-1", "HASH_FOR_cuda-toolkit-10-2_10.2.460-1_arm64.deb") # https://repo.download.nvidia.com/jetson/common/pool/main/c/cuda-toolkit-10-2/cuda-toolkit-10-2_10.2.460-1_arm64.deb
        ])
```

Reply:
I have pushed a completely non-functional WIP - please disregard all checks.

Does this approach/PR have any chance of making it through a review (i.e. an approach with an `AptPkgSource` - in `fancy toys.jl` or in separate `AptPkgSources.jl`/`Apt.jl` packages) or would the time be better spent to just add the boatload of FileSources that make up CUDA toolkit when fetched file by file from the APT repo.?

@maleadt Would this piggyback-on-APT be a nicer way of getting the NVIDIA binaries for the Linux-side of things  than the manual upload to Yggdrasil-approach? (I believe one can also get the cuda-cross packages like this)

Reply:
We can't get the Windows packages like this though, so we'll still have to maintain the current solution...

Note that NVIDIA is nowadays also distributing binaries in yet another way, https://developer.download.nvidia.com/compute/cuda/redist/, but that doesn't include semi-public stuff like CUDNN/CUTENSOR/TensorRT.

Reply:
The "fancy version" with an `AptPkgSource` (etc.) is now parked (indefinitely) at https://github.com/IHPSystems/Yggdrasil/tree/stemann/cuda_10.2_aarch64-linux-gnu_with_apt_pkg_source

Current version instead tries to use only FileSources.

Reply:
That's a shame, the Apt-based version was much nicer :slightly_smiling_face: Let's first merge this one so that we can get a working artifact for CUDA 10.2 on Aarch64 though.l

Reply:
> That's a shame, the Apt-based version was much nicer 🙂

Indeed, it was :-) Perhaps one day... :-)

@giordano This [Apt-approach](https://github.com/IHPSystems/Yggdrasil/tree/stemann/cuda_10.2_aarch64-linux-gnu_with_apt_pkg_source) for providing binaries, how would it fly?
1. Separate Apt.jl package, with a companion AptPkgSources.jl package to bridge Apt.jl to BinaryBuilder(Base) (at some point adding AptPkgSources to Yggdrasil build env.)
2. Add `AptPkgSource` and related types/methods BB/BBB

1, right?

Regarding Apt.jl: Simple pure-Julia version first (above link), later maybe wrap [libapt](https://github.com/Debian/apt) to get the dependency resolution properly done.